src/aviary/utils.py

```diff
@@ -370,17 +378,21 @@ def unsure_answer_letter(self) -> str | None:
 
     @property
     def question_prompt(self) -> str:
+        template_vars = {
+            "question": self.question,
+            "question_id": (
+                type(self).model_fields["question_id"].default
+                if self.prompt_without_id
+                else str(self.question_id)
+            ),
+        }
         if self.prompt_without_options:
-            return self.OPEN_ANSWER_PROMPT_TEMPLATE.format(
-                question=self.question,
-                question_id=self.question_id,
-            )
+            return self.OPEN_ANSWER_PROMPT_TEMPLATE.format(**template_vars)
```

Comment:
Is question_id used?

Reply:
Yeah it's used: https://github.com/Future-House/aviary/blob/v0.18.3/src/aviary/utils.py#L272